www/js/components/peek-point-card.js

```diff
@@ -1,30 +1,34 @@
-import React, {Component} from 'react';
+/*eslint-disable no-unused-vars*/
+import React, { Component } from 'react';
 import { Card, CardActions, CardText, RaisedButton } from 'material-ui';
 import PointHeader from './point-header';
+/*eslint-enable no-unused-vars*/
 
-const dayMap = ['Sunday', 'Monday', 'Tuesday', 'Wednesday',
-                'Thursday', 'Friday', 'Saturday'];
+const dayMap = [ 'Sunday', 'Monday', 'Tuesday', 'Wednesday',
+  'Thursday', 'Friday', 'Saturday' ];
 
 export class PeekPointCard extends Component {
 
-  getDays(seasons) {
-    let seasonDays = seasons[0].days;
-    const date = (new Date());
-    const currentMonth = date.getMonth();
-    const currentDate = date.getDate();
-    seasons.forEach((season)=>{
-      if ((season.seasonStart) && (season.seasonEnd) &&
-        (season.seasonStart.month <= currentMonth <= season.seasonEnd.month) &&
-        (season.seasonStart.date <= currentDate <= season.seasonEnd.date))
-      {
+  getDays( seasons ) {
+    let seasonDays = seasons[ 0 ].days;
+    const date = ( new Date() );
+    const curMonth = date.getMonth();
+    const curDate = date.getDate();
+    seasons.forEach( season => {
+      /*esfmt-ignore start*/
```

Comment:
Actually, I looked into this again. We don't need the ignores here, and in rating-point-card.js